docs/reference/extensions/django-framework.rst

```diff
@@ -43,6 +54,43 @@ application. In the following example we use it to specify ``libpq-dev``:
         # list required packages or slices for your Django application below.
         - libpq-dev
 
+.. _django-async-deps:
+
+``parts`` > ``django-framework/async-dependencies``
+===================================================
+
+In order to be able to use async Gunicorn workers you need to use
+``django-framework/async-dependencies`` part instead of
+``django-framework/dependencies`` part.
```

Comment:
I would rephrase this paragraph to say something like the following (maybe the last line is too long for the lint checks):
```
In order to use asynchronous Gunicorn workers, you need
to include ``django-framework/async-dependencies`` in the
``rockcraft.yaml`` while removing ``django-framework/dependencies``. 
```